### ISSUE TITLE
Multilined tabs - issue when moving tabs around

### ISSUE THREAD

**Issue by jmgau** [Aris-t2/CustomCSSforFx#74]
All of my tabs are at the bottom of the page. They work great. There only one small issue: I can't change the order of my tabs. They only move to one spot. So if I take my first tab on the first line and I try to move it to any other place, It will only move to the last spot on my second row for example.

Is there a solution to that situation ?

Thanks

**Comment by Aris-t2** [comment]
Have you read the template before posting this? You issue report has no info about your OS, OS version, Firefox version or any configuration.

I can move tabs without issues, if tabs toolbar is below main content. Make you run tests using the latest release version ([1.4.1](https://github.com/Aris-t2/CustomCSSforFx/releases/)).

Confirm you have enabled `tabs_below_navigation_toolbar.css` and `tabs_below_main_content.css`.

**Comment by jmgau** [comment]
Aris

First of all, I'm sorry for not giving all the details. 

To answer your question: Yes both tabs_below_navigation_toolbar.css and tabs_below_main_content.css are enabled. 
I tested some more and I found that I can't move the tabs around only when I enable : /* multirow / multi lined tabs -
I tried both (not at the same time) and I can't move tabs more than one or two spots to the right or left only.

If I disable the multirow everything works fine.

OS: Windows 10
OS version: 1703
OS theme; none
Firefox version: 57

Thanks for your time

[userChrome.zip](https://github.com/Aris-t2/CustomCSSforFx/files/1516219/userChrome.zip)
